contentcuration/contentcuration/frontend/shared/vuex/baseStore.js

```diff
@@ -19,7 +20,7 @@ export default function storeFactory(
     actions,
     getters,
     mutations,
-    plugins,
+    plugins: [ConnectionPlugin, ...(plugins || [])],
```

Comment:
At some point, e.g. if we wanted different `ConnectionPlugin` setups to be configurable, this could get pulled out of the defaults.